.github/ISSUE_TEMPLATE/discussion.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Discussion
+about: Delivery template
+title: "[Discussion]"
+labels: Discussion
+assignees: ''
+
```

Comment:
Add category `General` and change about value to `Entrega Semanal`, and title to `Resumen Semana al `